inspirehep/modules/authors/views.py

```diff
@@ -146,9 +147,11 @@ def validate():
     """Validate form and return validation errors."""
     if request.method != 'POST':
         abort(400)
+
+    is_update = True if request.args.get('is_update') == 'True' else False
```

Comment:
It's a bit tricky because until the form is submitted no workflow exists.

The only thing that you can do by modifying client side is to skip the validation. I think if I add validation on the server side on form submission then it would not be a big deal?